src/utils/event-sourcing/src/event_store.rs

```diff
@@ -44,6 +40,9 @@ pub type EventStream<'a, Event> = std::pin::Pin<
     Box<dyn tokio_stream::Stream<Item = Result<(EventID, Event), GetEventsError>> + Send + 'a>,
 >;
 
+pub type QueryStream<'a, Query> =
```

Comment:
Is not used any more?

Reply:
It was used, but I replaced on a more specialized `DatasetIDStream`